rpcs3/Emu/Cell/lv2/sys_rsx.cpp

```diff
@@ -536,12 +541,24 @@ error_code sys_rsx_context_attribute(u32 context_id, u32 package_id, u64 a3, u64
 	case 0x106: // ? (Used by cellGcmInitPerfMon)
 		break;
 
-	case 0x108: // cellGcmSetSecondVFrequency
+	case 0x108: // cellGcmSetVBlankFrequency, cellGcmSetSecondVFrequency
 		// a4 == 3, CELL_GCM_DISPLAY_FREQUENCY_59_94HZ
 		// a4 == 2, CELL_GCM_DISPLAY_FREQUENCY_SCANOUT
 		// a4 == 4, CELL_GCM_DISPLAY_FREQUENCY_DISABLE
-		// Note: Scanout/59_94 is ignored currently as we report refresh rate of 59_94hz as it is, so the difference doesnt matter
-		render->enable_second_vhandler.store(a4 != 4);
+
+		if (a5 == 1u)
+		{
+			// This function resets vsync state to enabled
+			render->requested_vsync = true;
```

Comment:
Where is the counterpart to disable this? We know PS3 had engines that had tearing when fps dropped below a threshold (e.g cryengine 2)

Reply:
I see sometimes in logs that the games reset it afterward and after cellVideoOutConfigure, I will add to it as well.

Reply:
Hmm, it is unlikely that they are calling videoOutConfigure during gameplay as that is a heavy operation. It is more likely to be toggled using this method. Is it known what happens if a5 is 0?

Reply:
I'm assuming that games also react to the unused CellVideoOutCallback. (maybe it's called when the display is changed)

Reply:
It's controlled by CellGcmSetFlipMode. You can choose 3 modes there, 1 for vsync, 1 for scanline sync and the last mode for all out anarchy. We only need to set vsync to engage when the first mode is chosen.

Reply:
> I'm assuming that games also react to the unused CellVideoOutCallback. (maybe it's called when the display is changed)

I've checked and no game uses cellVideoOutRegisterCallback.